### ISSUE TITLE
Certain BreadCraft foods don't have nutrients.

### ISSUE THREAD

**Issue by MagmaSlime123** [WesCook/Nutrition#247]
Title.

In #234, you added support for JAFFA as well as BreadCraft, however I noticed that certain BreadCraft foods don't have nutrients. Stuff like Breadcrumbs and possibly a few others, etc.

![2021-08-26_13 56 29](https://user-images.githubusercontent.com/51277766/131027484-e129fc32-1e80-4ed6-b8d8-dc282ca8d602.png)

**Comment by WesCook** [comment]
This left me feeling pretty confused.  As it turned out I never supported Breadcraft at all.  It's not mentioned in that bug reported, the linked release, or in the supported mods list.  The reason that some food items have support is because they share ore dictionary entries with existing food.  For example Baguette shares `bread` and `whitebreadlist`.

But since I already got out my IDE to investigate the problem, I went ahead and added support for the other foods.  Released in [v4.9.0](https://github.com/WesCook/Nutrition/releases/tag/v4.9.0) now.
